docs/protocols/java_object_graph_spec.md

```diff
@@ -53,8 +64,12 @@ If schema consistent mode is enabled globally or enabled for current class, clas
 
 - If class is registered, it will be written as a little-endian unsigned int: `class_id << 1` using fury unsigned int
```

Comment:
The concept of 'class_id' was not mentioned or defined elsewhere

Reply:
Added a description about class id:

Fury support register class by an optional id, the registration can be used to security check, and identify the class.
If the class is registered, the class will have a users provided or a auto-growing id unsigned int `class_id`.